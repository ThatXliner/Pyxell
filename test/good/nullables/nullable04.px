
for x in [4.0, null, -3.5] do
    print x
