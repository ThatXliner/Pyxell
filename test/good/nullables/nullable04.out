4.0
null
-3.5
